rust/src/config/cli.rs

```diff
@@ -33,9 +33,9 @@ impl Cli {
         opts.long_only(true);
         opts.optmulti(
             "",
-            "datadir",
-            "Set path for data directory",
-            DATA_DIR_OPTION_EXAMPLE
+            "gamedir",
```

Comment:
Same as with the JSON property, I would support both for at least one version...

Reply:
Done.